api/python/t4/search_util.py

```diff
@@ -112,3 +112,8 @@ def search(query, search_endpoint, limit, aws_region='us-east-1'):
                                     "bad query or a misconfigured search service.")
         setattr(exception, 'raw_response', raw_response)
         raise exception
+
+def get_raw_mapping_unpacked(endpoint, aws_region):
```

Comment:
Add the unpacking of the 'properties' fields here and return the cleaned up type info here instead of exposing these raw mappings.

Please also document the format of the elastic index dicts and include a link to relevant elastic docs.